data/hai/hai reveal 3 early.txt

```diff
@@ -429,4 +432,78 @@ event "role-play station active"
+				to display
+					has "FW Bounty 3: done"
+			`	"Such a vast supply is unlikely to have come from individual smugglers, so we must assume it's the result of a large-scale operation. I'm worried the Hai government may be arming pirates. If they are, I hope the weapons are just misguided tribute payments. In the south, we well know how useful bounties and tributes are when dealing with pirates, but from my time as a councilor, I can tell you the Republic will not be so understanding.`
+			`	"You must find out if the Hai government is arming the pirates and, if they are, put a stop to it. Don't tell them the Free Worlds asked you to do it and, whatever you do, don't let the Republic know. It could lead to an explosive diplomatic incident."`
+			label loop
+			choice
+				`	"A diplomat called Sayari told me she has a friend looking into 'interactions between Hai and pirates' who should contact me soon."`
+					goto byname
+					to display
+						not "Pirates with Hai Weapons 1: Katya's Message: explained"
+				`	"A trusted Hai contact of mine has friends that can help with the investigation. She also asked for discretion."`
+					goto discrete
+					to display
+						not "Pirates with Hai Weapons 1: Katya's Message: explained"
+				`	"That's worrying news. I'll look into it."`
+					goto acceptable
+					to display
+						not "Pirates with Hai Weapons 1: Katya's Message: explained"
+				`	"Don't worry. I'll stop the Hai from arming pirates."`
+					goto acceptable
+					to display
+						not "Pirates with Hai Weapons 1: Katya's Message: explained"
+				`	"I'll pursue that lead and see where it takes me."`
+					goto acceptable
+					to display
+						has "Pirates with Hai Weapons 1: Katya's Message: explained"
+				`	"If you vouch for her, then I'll trust her too."`
+					goto acceptable
+					to display
+						has "Pirates with Hai Weapons 1: Katya's Message: Katya vouched for Sayari"
+			label discrete
+			action
+				set "Pirates with Hai Weapons 1: Katya's Message: explained"
+			`	You explain about the note from a diplomat who also asked for "discretion."`
+			`	"A lead already! That sounds promising," says Katya.`
+				goto loop
+			label byname
+			action
+				set "Pirates with Hai Weapons 1: Katya's Message: explained"
+				set "Pirates with Hai Weapons 1: Katya's Message: Katya vouched for Sayari"
+			`	You explain about the note from Sayari who also asked for "discretion."`
+			`	Katya seems surprised. "Sayari! Can you describe her?"`
+			`	You explain Sayari's appearance to Katya and she responds, "Sayari has a long history of keeping peace between Hai and humans, longer than I've been alive. If she knows there are weapon deals, then I expect she wants to stop them. Still, be careful."`
```

Comment:
How does Katya know about Sayari?

Reply:
She's been a known entity to Humanity since first contact; assuming Katya was one of the people in the know beforehand, she should at least know of Sayari.

Reply:
I suppose, but if a Dirt Belt representative would know about the Hai, why don't the other hundreds or thousands of more influential representatives know? It might be too many people to keep a secret. I'll leave it for now, because Republic governance is a bigger issue.

Reply:
Sayari was one of the most famous, if not the most famous, Hai among human politicians until Hai Reveal. If you knew about the Hai, you knew she was the one who made first contact. It is an unforgettable moment in human history.

Also, Katya wasn't just *a* dirt belt representative. She was the *only* representative for many planets, and was also one of the founders of the Free Worlds government and deeply involved in secret things up until she was captured. If there were councilors who knew about the Hai, she certainly would have been one of them. Given that this is Parliament, which has Republic Intelligence working for them, it would require stupidity on an astounding scale, or some pretty big plot holes, for Parliament to not know about the Hai.

I'd say there's about a 99.993% chance Katya knew about Sayari.

Reply:
I know all this, but I was assuming that Katya would be much more involved and influential with Dirt Belt affairs (which sounded like a lot; piracy, famine, industrialisation, literacy, poverty, etc) than potential alien contacts, while politicians (a lot of them each planet, from the sounds of it) from highly populated northern planets like Prime, Sunracer, Shroud, etc. would certainly know. One Dirt Belt representative representing about a million people may not have the same amount of influence as those that represent billions of people. But I'll let this one go.